packages/migrations/src/actions/2025.04.29T00-00-00.organization-member-pagination.ts

```diff
@@ -0,0 +1,55 @@
+import type { MigrationExecutor } from '../pg-migrator';
+
+const date = new Date();
+
+const year = date.getFullYear();
+const month = date.getMonth() + 1;
+const day = date.getDate();
+
+const createdAt = `'${year}-${month}-${day}'`;
+
+export default {
+  name: '2025.04.29T00-00-00.organization-member-pagination.ts',
+  noTransaction: true,
+  // Adds a default role to OIDC integration and set index on "oidc_integrations"."default_role_id"
+  run: ({ sql }) => [
+    {
+      name: 'Add "organization_member"."created_at" column',
+      query: sql`
+        ALTER TABLE "organization_member"
+          ADD COLUMN IF NOT EXISTS "created_at" timestamptz NOT NULL DEFAULT ${sql.literalValue(createdAt)}::timestamp
```

Comment:
Does `::timestamp` auto format the createdAt literal? Because `date.getDate()` for example, will return `1` and not `01`

Reply:
Good question, I just double checked and it seems fine.

<img width="381" alt="image" src="https://github.com/user-attachments/assets/69911899-bc5f-482b-91c3-ab5ae71c4224" />